frontend/src/components/Patchnotes.jsx

```diff
@@ -27,74 +28,69 @@ const Patchnotes = ({ game }) => {
                     throw new Error('Network response was not ok');
                 }
                 const data = await response.json();
-                setPatchDetails(data);
+                console.log(data);
```

Comment:
did you mean to include this?

Reply:
no. i think i was testing something, and just forgot to delete it